linode/manager
# hotfix: Unescape encoding for user data

## Description 📝
Follow up to #9494. The previous PR fixed the Cloud crashing issue, but introduced another issue where we were not unescaping the encoded user data. So, when running the `atob` function on the user data, the output string included `%`s instead of just the original string.

reading:
https://base64.guru/developers/javascript/examples/unicode-strings
https://stackoverflow.com/questions/30631927/converting-to-base64-in-javascript-without-deprecated-escape-call (see the second answer by `T S`)

## Preview 📷
| Before  | After   |
| ------- | ------- |
| ![before](https://github.com/linode/manager/assets/115299789/02e607e0-84d9-49a2-bbe0-1d183195e988) | ![after](https://github.com/linode/manager/assets/115299789/f270f978-9e62-4058-8447-bcbe1ed533fc) |

## How to test 🧪
- Add a console log after line 706 to log the user data `console.log(utoa(this.props.userData))`
- Follow the steps in #9494 
- Take the result of the console log and run `atob`
- The result should be readable and not have unexpected characters such as `%`
